protocols.json

```diff
@@ -18,7 +18,6 @@
     "008-PtEdoTez"
   ],
   "active": [
-    "008-PtEdo2Zk",
```

Comment:
Please move to the `allowed` instead of removing, so that 008 is still compiled in the `tezos-node` binary (it simplifies bootstrap process for the `mainnet`)